libsplinter/Cargo.toml

```diff
@@ -110,6 +110,7 @@ experimental = [
     "registry-database",
     "service-arg-validation",
     "service-network",
+    "sqlite-admin-service-event-store",
```

Comment:
In other uses of guarded implementations, we've used the main feature, and then added the subtype.  So this should probably be: `admin-service-event-store-sqlite`.

Though, in the sqlite case, we haven't been guarded that on its own, as it can be unit tested, unlike the postgres-backed store.

Reply:
I changed the other feature added to `admin-service-event-store-diesel` and put the sqlite test (originally behind `sqlite-admin-service-event-store`) is now guarded by: `#[cfg(all(test, feature = "admin-service-event-store-diesel", feature = "sqlite"))]`